dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/registry/WorkerWaitingStrategy.java

```diff
@@ -52,6 +53,16 @@ public class WorkerWaitingStrategy implements WorkerConnectStrategy {
     @Autowired
     private WorkerTaskExecutorThreadPool workerManagerThread;
 
+    public WorkerWaitingStrategy(@NonNull WorkerConfig workerConfig,
```

Comment:
hi @pegasas Through the @Service annotation, I judge that this class is a singleton and injected into the spring container, so is this constructor necessary? Where is it used? Is there anything I haven't noticed?